views/clubs.handlebars

```diff
@@ -32,4 +33,5 @@
             </div>
         {{/if}}
     </div>
+    <a href="/createclub" class="btn btn-primary btn-lg su">Create a club here!</a>
```

Comment:
We should only show this button if the user is logged in. If you look that my PR, there's a handlebars if that shows the "Login or signup" if the user isn't logged in and the Create Club button if it is. Here's an example that does this with the "Gravitate" button...
https://github.com/nobleburgundy/orbit/blob/10af15a6aad777777e450c23d0545f0b84dd58bc/views/club_page.handlebars#L22